server/dash/dashboards/neighborhood.py

```diff
@@ -43,35 +47,67 @@ def populate_options():
 # Define callback to update graph
 @app.callback(
     Output('graph1', 'figure'),
-    [Input("council_list", "value")]
+    [Input("council_list", "value"),
+    Input("data_type", "value")]
 )
-def update_figure(selected_council):
-
+def update_figure(selected_council, selected_timeframe):
     neighborhood_sum_df = df[df.council_name == selected_council].groupby(['created_date']).agg('sum').reset_index()  # noqa
     total_sum_df = df.groupby(['created_date']).agg('sum').reset_index()
-    total_sum_df["nc_avg"] = total_sum_df["counts"] / 99
-    merged_df = neighborhood_sum_df.merge(total_sum_df["nc_avg"].to_frame(), left_index=True, right_index=True)  # noqa
 
+    total_sum_df['nc_ma'] = total_sum_df.counts.rolling(selected_timeframe, center=True).mean()/NEIGHBORHOOD_COUNCILS_NUMBER
+    neighborhood_sum_df['select_nc_ma'] = neighborhood_sum_df.counts.rolling(selected_timeframe, center=True).mean()
+    
+    merged_df = pd.merge(neighborhood_sum_df, total_sum_df, on=["created_date", "created_date"])
+    
     fig = px.line(
         merged_df,
         x="created_date",
-        y=['counts', 'nc_avg'],
+        y=['select_nc_ma', 'nc_ma'],
         color_discrete_sequence=DISCRETE_COLORS,
-        labels=LABELS,
-        title="Comparison trend for " + selected_council
+        labels={
+            "created_date": "311 Request Date",
+            "value": "Total Requests"
+                },
+        title=selected_council + " vs 99 Neighborhood Councils 311 Requests Average"
+    )
+    newnames = {
+        "select_nc_ma": selected_council, 
+        "nc_ma": "99 Neighborhood Councils Average"
+    }
+    fig.for_each_trace(lambda t: t.update(name = newnames[t.name],
+                                      legendgroup = newnames[t.name],
+                                      hovertemplate = t.hovertemplate.replace(t.name, newnames[t.name])
+                                     )
     )
 
     fig.update_xaxes(
         tickformat="%a\n%m/%d",
     )
 
+    fig.update_yaxes(
+        rangemode="tozero"
+    )
+
     fig.update_traces(
         mode='markers+lines'
     )  # add markers to lines
 
     apply_figure_style(fig)
 
     return fig
+"""Creates comparison graph
+
+
+    Args:
+        selected_council: The selected council from the drop down menu
+        selected_timeframe: radio button selection of time window for computing moving average
+
+
+    Returns:
+        Graph that compares the selected moving average between the chosen council and the moving average of the avg of all
+        the 99 neighborhood councils 
+
+"""
```

Comment:
Please move this to be right under `def update_figure...`

Please add periods at the end of each sentence and include datatypes for the Args and Returns.

Reply:
I changed it, let me know if the data type are correct